Sources/Datadog/RUM/RUMMonitor/RUMCommand.swift

```diff
@@ -281,3 +281,10 @@ internal struct RUMAddLongTaskCommand: RUMCommand {
 
     let duration: TimeInterval
 }
+
+// MARK: - RUM Web Events related commands
+
+internal struct RUMWebViewCommand: RUMCommand {
```

Comment:
We should call it with _verb + noun_, like other commands. Now `RUMWebViewCommand` doesn't tell what it does and it has no comment indicating it. As I understand, we may want to name it as `RUMKeepSessionAliveCommand` or similar. Additional comment on why this command was introduced can be handy for next readers, especially that we don't handle this command in any scope.